src/include/os.h

```diff
@@ -1,12 +1,23 @@
 #ifndef __OS_H_
 #define __OS_H_
 
+#include <stdint.h>
+
 int GetTID(void);
 int SchedYield(void);
 void EmitSignal(void* emu, int sig, void* addr, int code);
 void EmitDiv0(void* emu, void* addr, int code);
 void EmitInterruption(void* emu, int num, void* addr);
 
+// These functions only applies to Linux
+int IsBridgeSignature(char s, char c);
+int IsNativeCall(uintptr_t addr, int is32bits, uintptr_t* calladdress, uint16_t* retn);
+void NativeCall(void* emu, void* addr);
+void* EmuFork(void* emu, int forktype);
+
+void EmuX64Syscall(void* emu);
```

Comment:
I am sure  Syscall also exist on Windows.

Reply:
And Wine handle it, at least on the 64bits side. I think it's ignored on 32bits

Reply:
yeah, that's why there is a newline from the above functions, but I should clarify the scope of the comment.